src/utils/formValidation.ts

```diff
@@ -1,6 +1,40 @@
 export const mustBePresent = (value: any): string | undefined =>
   value ? undefined : 'validation.message.required';
```

Comment:
Since we are improving the validation in this PR, should we consider also whitespaces invalid values?

Reply:
Good idea